changes_test.go

```diff
@@ -390,12 +390,14 @@ var fromDataTests = []struct {
 			ContainerType: "lxc",
 			Series:        "trusty",
 			ParentId:      "$addMachines-6",
+			Constraints:   "cpu-cores=4 cpu-power=42",
```

Comment:
We could a test for when constraints are specified on the application and the unit is located to "new".